server/src/main/java/org/elasticsearch/TransportVersion.java

```diff
@@ -0,0 +1,249 @@
+    public static final TransportVersion V_7_13_2 = new TransportVersion(7_13_02_92, "2a6fc74c-4c44-4264-a619-37437cd2c5a0");
+    public static final TransportVersion V_7_13_3 = new TransportVersion(7_13_03_92, "a31592f5-f8d2-490c-a02e-da9501823d8d");
+    public static final TransportVersion V_7_13_4 = new TransportVersion(7_13_04_92, "3143240d-1831-4186-8a19-963336c4cea0");
+    public static final TransportVersion V_7_14_0 = new TransportVersion(7_14_00_90, "8cf0954c-b085-467f-b20b-3cb4b2e69e3e");
+    public static final TransportVersion V_7_14_1 = new TransportVersion(7_14_01_90, "3dbb62c3-cf73-4c76-8d5a-4ca70afe2c70");
+    public static final TransportVersion V_7_14_2 = new TransportVersion(7_14_02_90, "7943ae20-df60-45e5-97ba-82fc0dfc8b89");
+    public static final TransportVersion V_7_15_0 = new TransportVersion(7_15_00_90, "2273ac0e-00bb-4024-9e2e-ab78981623c6");
+    public static final TransportVersion V_7_15_1 = new TransportVersion(7_15_01_90, "a8c3503d-3452-45cf-b385-e855e16547fe");
+    public static final TransportVersion V_7_15_2 = new TransportVersion(7_15_02_90, "fbb8ad69-02e2-4c90-b2e4-23947107f8b4\n");
+    public static final TransportVersion V_7_16_0 = new TransportVersion(7_16_00_9_1, "59abadd2-25db-4547-a991-c92306a3934e");
+    public static final TransportVersion V_7_16_1 = new TransportVersion(7_16_01_9_1, "4ace6b6b-8bba-427f-8755-9e3b40092138");
+    public static final TransportVersion V_7_16_2 = new TransportVersion(7_16_02_9_1, "785567b9-b320-48ef-b538-1753228904cd");
+    public static final TransportVersion V_7_16_3 = new TransportVersion(7_16_03_9_1, "facf5ae7-3d4e-479c-9142-72529b784e30");
+    public static final TransportVersion V_7_17_0 = new TransportVersion(7_17_00_9_1, "322efe93-4c73-4e15-9274-bb76836c8fa8");
+    public static final TransportVersion V_7_17_1 = new TransportVersion(7_17_01_9_1, "51c72842-7974-4669-ad25-bf13ba307307");
+    public static final TransportVersion V_7_17_2 = new TransportVersion(7_17_02_9_1, "82bea8d0-bfea-47c2-b7d3-217d8feb67e3");
+    public static final TransportVersion V_7_17_3 = new TransportVersion(7_17_03_9_1, "a909c2f4-5cb8-46bf-af0f-cd18d1b7e9d2");
+    public static final TransportVersion V_7_17_4 = new TransportVersion(7_17_04_9_1, "5076e164-18a4-4373-8be7-15f1843c46db");
+    public static final TransportVersion V_7_17_5 = new TransportVersion(7_17_05_9_1, "da7e3509-7f61-4dd2-8d23-a61f628a62f6");
+    public static final TransportVersion V_7_17_6 = new TransportVersion(7_17_06_9_1, "a47ecf02-e457-474f-887d-ee15a7ebd969");
+    public static final TransportVersion V_7_17_7 = new TransportVersion(7_17_07_9_1, "108ba576-bb28-42f4-bcbf-845a0ce52560");
+    public static final TransportVersion V_7_17_8 = new TransportVersion(7_17_08_9_1, "82a3e70d-cf0e-4efb-ad16-6077ab9fe19f");
+    public static final TransportVersion V_7_17_9 = new TransportVersion(7_17_09_9_1, "afd50dda-735f-4eae-9309-3218ffec1b2d");
```

Comment:
is formatting of numbers off? 7_17_08_9_1 vs 7_17_08_91
and perhaps it should be 7_17_08_00 so that we get ourselves a 100 of transport versions to use

Reply:
Oh yes, must have screwed up the copy/paste from Version